packages/@aws-cdk-testing/framework-integ/test/aws-codebuild/test/integ.project-fleet-attribute-based-compute.ts

```diff
@@ -38,6 +89,7 @@ class FleetStack extends cdk.Stack {
       }),
       environment: {
         buildImage: codebuild.LinuxBuildImage.STANDARD_7_0,
+        fleet: this.fleet,
```

Comment:
@badmintoncryer when you [added](https://github.com/aws/aws-cdk/pull/32251) this integration test earlier this year, you weren't actually using the fleet in the test 🥲. It works now!

Reply:
@badmintoncryer could I trouble you for a review?  I think you count as a "community reviewer" and this PR has been waiting for one for many months now :(.